src/pages/index.tsx

```diff
@@ -235,10 +236,9 @@ const Home: NextPage<HomePageProps> = ({ storefront }) => {
             <>Loading</>
           ) : (
             <ul className="grid grid-cols-4 gap-6">
-              {nfts.data?.nfts.map(({ name, details, address }) => (
-                <li key={address}>
-                  <img src={details?.image as string} alt="nft image" className="aspect-square rounded-lg" />
-                  {name}
+              {nfts.data?.nfts.map((n) => (
+                <li key={n.address}> 
+                  <NftCard address={n.address} nft={n}/>
```

Comment:
Since Nft cards are only rendered on this page I advocate for having the html inline. Pull into component when used cross page.

Reply:
Okay, no problem!